rllib/policy/policy.py

```diff
@@ -211,7 +221,7 @@ def get_weights(self):
         Returns:
             weights (obj): Serializable copy or view of model weights
         """
-        raise NotImplementedError
+        pass
```

Comment:
Why make this change?

Reply:
I don't want to make this an `@abstractmethod`, b/c then we have to implement it (using probably just `pass` anyway) for all our "dummy" Policies that we use for testing.

Reply:
Like: TestPolicy, StatefulPolicy, MockPolicy, etc..

Reply:
ah, I mean why not keep it as a raise?

Reply:
Then the new ABCMeta metaclass of the base Policy would require this method to be implemented (raise or not) in all the above dummies.

Reply:
Actually, you are right, we could keep the `raise` here and still have it a non `@abstractmethod`. I'll change it back in a future PR.